src/eom-util.c

```diff
@@ -427,3 +427,47 @@ eom_util_show_file_in_filemanager (GFile *file, GtkWindow *toplevel)
 	if (!done)
 		_eom_util_show_file_in_filemanager_fallback (file, toplevel);
 }
+
+gboolean
+eom_util_dialog_page_scroll_event_cb (GtkWidget*			widget,
+									  GdkEventScroll*		event)
```

Comment:
fix code formatting

Reply:
Indentation!
Should be look like this:
```
gboolean
eom_util_dialog_page_scroll_event_cb (GtkWidget*       widget,
                                      GdkEventScroll*  event)
```

Reply:
Sorru, i miss this line in my previous commit. Done now.